integration/rootless: should query for known pods

**Which feature do you think can be improved?**

The `integration/rootless/rootless_test.sh` test.

**How can it be improved?**

That test is going to check for the pod being ready by issuing the command:

`waitForProcess 15 3 "sudo crictl inspectp $(sudo crictl pods -q) | jq '.status.state' | grep 'SANDBOX_READY'"`

Notice that `sudo crictl pods -q` is going to query for all running pods, that includes residual pods from previous executed tests which failed to clean up the environment. As a result the rootless test can fail if there are leftovers pods.

I can think on the following possible solutions:
1. Start the test's pod on its own namespace (i.e avoid `default`) and query for the pods from that specific namespace
2. Rename the pod to `something-rootless` and query by name
3. Get the PID returned when the pod was created.   

**Additional Information**

Recently the job [kata-containers-2.0-tests-centos-PR#1160](http://jenkins.katacontainers.io/job/kata-containers-2.0-tests-centos-PR/1160/consoleFull) for one of my PRs failed because of the rootless test.

After inspecting the log (see below) it seems that there was a leftover from a previous test and that's making the rootless test to fail. On the snippet below you can see that two pods were stopped and removed but in reality only the _41f9b2fc826c9e95301196_ was actually started by the test; so the _8e56e4946042fbe5dce8609ea349724e68f_ is probably a leftover.
```
15:13:12 Stopped sandbox 41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24
15:13:12 Stopped sandbox 8e56e4946042fbe5dce8609ea349724e68f1a70eb0d7d2839a32f07787bc236c
15:13:12 Removed sandbox 41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24
15:13:12 Removed sandbox 8e56e4946042fbe5dce8609ea349724e68f1a70eb0d7d2839a32f07787bc236c
```

The rootless test full log:
```
15:12:52 INFO: Running rootless tests
15:12:52 bash -f integration/rootless/rootless_test.sh
15:12:56 41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24
15:12:56 {
15:12:56   "status": {
15:12:56     "id": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:12:56     "metadata": {
15:12:56       "attempt": 1,
15:12:56       "name": "nginx-sandbox",
15:12:56       "namespace": "default",
15:12:56       "uid": "hdishd83djaidwnduwk28bcs3"
15:12:56     },
15:12:56     "state": "SANDBOX_READY",
15:12:56     "createdAt": "2021-10-25T18:12:52.859870345Z",
15:12:56     "network": {
15:12:56       "ip": "10.244.0.2"
15:12:56     },
15:12:56     "linux": {
15:12:56       "namespaces": {
15:12:56         "options": {
15:12:56           "ipc": "POD",
15:12:56           "network": "POD",
15:12:56           "pid": "POD"
15:12:56         }
15:12:56       }
15:12:56     },
15:12:56     "labels": {},
15:12:56     "annotations": {}
15:12:56   },
15:12:56   "info": {
15:12:56     "image": "k8s.gcr.io/pause:3.5",
15:12:56     "pid": 108390,
15:12:56     "runtimeSpec": {
15:12:56       "ociVersion": "1.0.2-dev",
15:12:56       "process": {
15:12:56         "user": {
15:12:56           "uid": 0,
15:12:56           "gid": 0
15:12:56         },
15:12:56         "args": [
15:12:56           "/pause"
15:12:56         ],
15:12:56         "env": [
15:12:56           "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
15:12:56           "TERM=xterm"
15:12:56         ],
15:12:56         "cwd": "/",
15:12:56         "capabilities": {
15:12:56           "bounding": [
15:12:56             "CAP_CHOWN",
15:12:56             "CAP_DAC_OVERRIDE",
15:12:56             "CAP_FSETID",
15:12:56             "CAP_FOWNER",
15:12:56             "CAP_SETGID",
15:12:56             "CAP_SETUID",
15:12:56             "CAP_SETPCAP",
15:12:56             "CAP_NET_BIND_SERVICE",
15:12:56             "CAP_KILL"
15:12:56           ],
15:12:56           "effective": [
15:12:56             "CAP_CHOWN",
15:12:56             "CAP_DAC_OVERRIDE",
15:12:56             "CAP_FSETID",
15:12:56             "CAP_FOWNER",
15:12:56             "CAP_SETGID",
15:12:56             "CAP_SETUID",
15:12:56             "CAP_SETPCAP",
15:12:56             "CAP_NET_BIND_SERVICE",
15:12:56             "CAP_KILL"
15:12:56           ],
15:12:56           "inheritable": [
15:12:56             "CAP_CHOWN",
15:12:56             "CAP_DAC_OVERRIDE",
15:12:56             "CAP_FSETID",
15:12:56             "CAP_FOWNER",
15:12:56             "CAP_SETGID",
15:12:56             "CAP_SETUID",
15:12:56             "CAP_SETPCAP",
15:12:56             "CAP_NET_BIND_SERVICE",
15:12:56             "CAP_KILL"
15:12:56           ],
15:12:56           "permitted": [
15:12:56             "CAP_CHOWN",
15:12:56             "CAP_DAC_OVERRIDE",
15:12:56             "CAP_FSETID",
15:12:56             "CAP_FOWNER",
15:12:56             "CAP_SETGID",
15:12:56             "CAP_SETUID",
15:12:56             "CAP_SETPCAP",
15:12:56             "CAP_NET_BIND_SERVICE",
15:12:56             "CAP_KILL"
15:12:56           ]
15:12:56         },
15:12:56         "oomScoreAdj": -998,
15:12:56         "selinuxLabel": "system_u:system_r:container_kvm_t:s0:c288,c583"
15:12:56       },
15:12:56       "root": {
15:12:56         "path": "/var/lib/containers/storage/overlay/e082e3f9c46e18e50e53b6a49c1739cfd029452c469f07a7937794a5a50b9db0/merged",
15:12:56         "readonly": true
15:12:56       },
15:12:56       "hostname": "41f9b2fc826c",
15:12:56       "mounts": [
15:12:56         {
15:12:56           "destination": "/proc",
15:12:56           "type": "proc",
15:12:56           "source": "proc",
15:12:56           "options": [
15:12:56             "nosuid",
15:12:56             "noexec",
15:12:56             "nodev"
15:12:56           ]
15:12:56         },
15:12:56         {
15:12:56           "destination": "/dev",
15:12:56           "type": "tmpfs",
15:12:56           "source": "tmpfs",
15:12:56           "options": [
15:12:56             "nosuid",
15:12:56             "strictatime",
15:12:56             "mode=755",
15:12:56             "size=65536k"
15:12:56           ]
15:12:56         },
15:12:56         {
15:12:56           "destination": "/dev/pts",
15:12:56           "type": "devpts",
15:12:56           "source": "devpts",
15:12:56           "options": [
15:12:56             "nosuid",
15:12:56             "noexec",
15:12:56             "newinstance",
15:12:56             "ptmxmode=0666",
15:12:56             "mode=0620",
15:12:56             "gid=5"
15:12:56           ]
15:12:56         },
15:12:56         {
15:12:56           "destination": "/dev/mqueue",
15:12:56           "type": "mqueue",
15:12:56           "source": "mqueue",
15:12:56           "options": [
15:12:56             "nosuid",
15:12:56             "noexec",
15:12:56             "nodev"
15:12:56           ]
15:12:56         },
15:12:56         {
15:12:56           "destination": "/sys",
15:12:56           "type": "sysfs",
15:12:56           "source": "sysfs",
15:12:56           "options": [
15:12:56             "nosuid",
15:12:56             "noexec",
15:12:56             "nodev",
15:12:56             "ro"
15:12:56           ]
15:12:56         },
15:12:56         {
15:12:56           "destination": "/dev/shm",
15:12:56           "type": "bind",
15:12:56           "source": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/shm",
15:12:56           "options": [
15:12:56             "rw",
15:12:56             "bind"
15:12:56           ]
15:12:56         },
15:12:56         {
15:12:56           "destination": "/etc/hostname",
15:12:56           "type": "bind",
15:12:56           "source": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/hostname",
15:12:56           "options": [
15:12:56             "ro",
15:12:56             "bind",
15:12:56             "nodev",
15:12:56             "nosuid",
15:12:56             "noexec"
15:12:56           ]
15:12:56         }
15:12:56       ],
15:12:56       "annotations": {
15:12:56         "io.kubernetes.cri-o.Labels": "null",
15:12:56         "io.kubernetes.cri-o.Image": "k8s.gcr.io/pause:3.5",
15:12:56         "io.kubernetes.cri-o.ContainerID": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:12:56         "io.kubernetes.cri-o.RuntimeHandler": "kata",
15:12:56         "io.kubernetes.cri-o.HostNetwork": "false",
15:12:56         "io.kubernetes.cri-o.IP.0": "10.244.0.2",
15:12:56         "io.kubernetes.cri-o.HostName": "41f9b2fc826c",
15:12:56         "io.kubernetes.cri-o.HostnamePath": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/hostname",
15:12:56         "io.kubernetes.cri-o.ContainerName": "k8s_POD_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1",
15:12:56         "io.kubernetes.cri-o.PrivilegedRuntime": "false",
15:12:56         "io.kubernetes.cri-o.ResolvPath": "",
15:12:56         "io.kubernetes.cri-o.MountPoint": "/var/lib/containers/storage/overlay/e082e3f9c46e18e50e53b6a49c1739cfd029452c469f07a7937794a5a50b9db0/merged",
15:12:56         "io.kubernetes.cri-o.CNIResult": "{\"cniVersion\":\"0.4.0\",\"interfaces\":[{\"name\":\"cni0\",\"mac\":\"ae:b0:19:cf:37:45\"},{\"name\":\"veth2325b942\",\"mac\":\"ba:dc:1e:4e:05:f6\"},{\"name\":\"eth0\",\"mac\":\"da:81:56:36:25:34\",\"sandbox\":\"/var/run/netns/0cc4fc54-1d0d-460e-8509-822548ce4a9c\"}],\"ips\":[{\"version\":\"4\",\"interface\":2,\"address\":\"10.244.0.2/24\",\"gateway\":\"10.244.0.1\"}],\"routes\":[{\"dst\":\"10.244.0.0/16\"},{\"dst\":\"0.0.0.0/0\",\"gw\":\"10.244.0.1\"}],\"dns\":{}}",
15:12:56         "io.kubernetes.cri-o.Metadata": "{\"Name\":\"nginx-sandbox\",\"UID\":\"hdishd83djaidwnduwk28bcs3\",\"Namespace\":\"default\",\"Attempt\":1}",
15:12:56         "io.kubernetes.cri-o.Namespace": "default",
15:12:56         "io.kubernetes.cri-o.SandboxID": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:12:56         "io.kubernetes.cri-o.KubeName": "nginx-sandbox",
15:12:56         "io.container.manager": "cri-o",
15:12:56         "io.kubernetes.cri-o.PortMappings": "[]",
15:12:56         "io.kubernetes.cri-o.Name": "k8s_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1",
15:12:56         "io.kubernetes.cri-o.ContainerType": "sandbox",
15:12:56         "io.kubernetes.cri-o.ShmPath": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/shm",
15:12:56         "io.kubernetes.cri-o.CgroupParent": "",
15:12:56         "io.kubernetes.cri-o.LogPath": "/tmp/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24.log",
15:12:56         "io.kubernetes.cri-o.NamespaceOptions": "{}",
15:12:56         "io.kubernetes.cri-o.Created": "2021-10-25T18:12:52.859870345Z",
15:12:56         "io.kubernetes.cri-o.SeccompProfilePath": "",
15:12:56         "io.kubernetes.cri-o.Annotations": "null",
15:12:56         "org.systemd.property.CollectMode": "'inactive-or-failed'"
15:12:56       },
15:12:56       "linux": {
15:12:56         "resources": {
15:12:56           "devices": [
15:12:56             {
15:12:56               "allow": false,
15:12:56               "access": "rwm"
15:12:56             }
15:12:56           ],
15:12:56           "cpu": {
15:12:56             "shares": 2
15:12:56           }
15:12:56         },
15:12:56         "namespaces": [
15:12:56           {
15:12:56             "type": "pid"
15:12:56           },
15:12:56           {
15:12:56             "type": "network",
15:12:56             "path": "/var/run/netns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:12:56           },
15:12:56           {
15:12:56             "type": "ipc",
15:12:56             "path": "/var/run/ipcns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:12:56           },
15:12:56           {
15:12:56             "type": "uts",
15:12:56             "path": "/var/run/utsns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:12:56           },
15:12:56           {
15:12:56             "type": "mount"
15:12:56           }
15:12:56         ],
15:12:56         "mountLabel": "system_u:object_r:container_file_t:s0:c288,c583"
15:12:56       }
15:12:56     }
15:12:56   }
15:12:56 }
15:12:56 /tmp/jenkins/workspace/kata-containers-2.0-tests-centos-PR/go/src/github.com/kata-containers/tests/lib/common.bash: line 61: 8e56e4946042fbe5dce8609ea349724e68f1a70eb0d7d2839a32f07787bc236c: command not found
15:12:59 {
15:12:59   "status": {
15:12:59     "id": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:12:59     "metadata": {
15:12:59       "attempt": 1,
15:12:59       "name": "nginx-sandbox",
15:12:59       "namespace": "default",
15:12:59       "uid": "hdishd83djaidwnduwk28bcs3"
15:12:59     },
15:12:59     "state": "SANDBOX_READY",
15:12:59     "createdAt": "2021-10-25T18:12:52.859870345Z",
15:12:59     "network": {
15:12:59       "ip": "10.244.0.2"
15:12:59     },
15:12:59     "linux": {
15:12:59       "namespaces": {
15:12:59         "options": {
15:12:59           "ipc": "POD",
15:12:59           "network": "POD",
15:12:59           "pid": "POD"
15:12:59         }
15:12:59       }
15:12:59     },
15:12:59     "labels": {},
15:12:59     "annotations": {}
15:12:59   },
15:12:59   "info": {
15:12:59     "image": "k8s.gcr.io/pause:3.5",
15:12:59     "pid": 108390,
15:12:59     "runtimeSpec": {
15:12:59       "ociVersion": "1.0.2-dev",
15:12:59       "process": {
15:12:59         "user": {
15:12:59           "uid": 0,
15:12:59           "gid": 0
15:12:59         },
15:12:59         "args": [
15:12:59           "/pause"
15:12:59         ],
15:12:59         "env": [
15:12:59           "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
15:12:59           "TERM=xterm"
15:12:59         ],
15:12:59         "cwd": "/",
15:12:59         "capabilities": {
15:12:59           "bounding": [
15:12:59             "CAP_CHOWN",
15:12:59             "CAP_DAC_OVERRIDE",
15:12:59             "CAP_FSETID",
15:12:59             "CAP_FOWNER",
15:12:59             "CAP_SETGID",
15:12:59             "CAP_SETUID",
15:12:59             "CAP_SETPCAP",
15:12:59             "CAP_NET_BIND_SERVICE",
15:12:59             "CAP_KILL"
15:12:59           ],
15:12:59           "effective": [
15:12:59             "CAP_CHOWN",
15:12:59             "CAP_DAC_OVERRIDE",
15:12:59             "CAP_FSETID",
15:12:59             "CAP_FOWNER",
15:12:59             "CAP_SETGID",
15:12:59             "CAP_SETUID",
15:12:59             "CAP_SETPCAP",
15:12:59             "CAP_NET_BIND_SERVICE",
15:12:59             "CAP_KILL"
15:12:59           ],
15:12:59           "inheritable": [
15:12:59             "CAP_CHOWN",
15:12:59             "CAP_DAC_OVERRIDE",
15:12:59             "CAP_FSETID",
15:12:59             "CAP_FOWNER",
15:12:59             "CAP_SETGID",
15:12:59             "CAP_SETUID",
15:12:59             "CAP_SETPCAP",
15:12:59             "CAP_NET_BIND_SERVICE",
15:12:59             "CAP_KILL"
15:12:59           ],
15:12:59           "permitted": [
15:12:59             "CAP_CHOWN",
15:12:59             "CAP_DAC_OVERRIDE",
15:12:59             "CAP_FSETID",
15:12:59             "CAP_FOWNER",
15:12:59             "CAP_SETGID",
15:12:59             "CAP_SETUID",
15:12:59             "CAP_SETPCAP",
15:12:59             "CAP_NET_BIND_SERVICE",
15:12:59             "CAP_KILL"
15:12:59           ]
15:12:59         },
15:12:59         "oomScoreAdj": -998,
15:12:59         "selinuxLabel": "system_u:system_r:container_kvm_t:s0:c288,c583"
15:12:59       },
15:12:59       "root": {
15:12:59         "path": "/var/lib/containers/storage/overlay/e082e3f9c46e18e50e53b6a49c1739cfd029452c469f07a7937794a5a50b9db0/merged",
15:12:59         "readonly": true
15:12:59       },
15:12:59       "hostname": "41f9b2fc826c",
15:12:59       "mounts": [
15:12:59         {
15:12:59           "destination": "/proc",
15:12:59           "type": "proc",
15:12:59           "source": "proc",
15:12:59           "options": [
15:12:59             "nosuid",
15:12:59             "noexec",
15:12:59             "nodev"
15:12:59           ]
15:12:59         },
15:12:59         {
15:12:59           "destination": "/dev",
15:12:59           "type": "tmpfs",
15:12:59           "source": "tmpfs",
15:12:59           "options": [
15:12:59             "nosuid",
15:12:59             "strictatime",
15:12:59             "mode=755",
15:12:59             "size=65536k"
15:12:59           ]
15:12:59         },
15:12:59         {
15:12:59           "destination": "/dev/pts",
15:12:59           "type": "devpts",
15:12:59           "source": "devpts",
15:12:59           "options": [
15:12:59             "nosuid",
15:12:59             "noexec",
15:12:59             "newinstance",
15:12:59             "ptmxmode=0666",
15:12:59             "mode=0620",
15:12:59             "gid=5"
15:12:59           ]
15:12:59         },
15:12:59         {
15:12:59           "destination": "/dev/mqueue",
15:12:59           "type": "mqueue",
15:12:59           "source": "mqueue",
15:12:59           "options": [
15:12:59             "nosuid",
15:12:59             "noexec",
15:12:59             "nodev"
15:12:59           ]
15:12:59         },
15:12:59         {
15:12:59           "destination": "/sys",
15:12:59           "type": "sysfs",
15:12:59           "source": "sysfs",
15:12:59           "options": [
15:12:59             "nosuid",
15:12:59             "noexec",
15:12:59             "nodev",
15:12:59             "ro"
15:12:59           ]
15:12:59         },
15:12:59         {
15:12:59           "destination": "/dev/shm",
15:12:59           "type": "bind",
15:12:59           "source": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/shm",
15:12:59           "options": [
15:12:59             "rw",
15:12:59             "bind"
15:12:59           ]
15:12:59         },
15:12:59         {
15:12:59           "destination": "/etc/hostname",
15:12:59           "type": "bind",
15:12:59           "source": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/hostname",
15:12:59           "options": [
15:12:59             "ro",
15:12:59             "bind",
15:12:59             "nodev",
15:12:59             "nosuid",
15:12:59             "noexec"
15:12:59           ]
15:12:59         }
15:12:59       ],
15:12:59       "annotations": {
15:12:59         "io.kubernetes.cri-o.LogPath": "/tmp/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24.log",
15:12:59         "io.kubernetes.cri-o.NamespaceOptions": "{}",
15:12:59         "io.kubernetes.cri-o.Created": "2021-10-25T18:12:52.859870345Z",
15:12:59         "io.kubernetes.cri-o.SeccompProfilePath": "",
15:12:59         "io.kubernetes.cri-o.Annotations": "null",
15:12:59         "org.systemd.property.CollectMode": "'inactive-or-failed'",
15:12:59         "io.kubernetes.cri-o.IP.0": "10.244.0.2",
15:12:59         "io.kubernetes.cri-o.Labels": "null",
15:12:59         "io.kubernetes.cri-o.Image": "k8s.gcr.io/pause:3.5",
15:12:59         "io.kubernetes.cri-o.ContainerID": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:12:59         "io.kubernetes.cri-o.RuntimeHandler": "kata",
15:12:59         "io.kubernetes.cri-o.HostNetwork": "false",
15:12:59         "io.kubernetes.cri-o.HostName": "41f9b2fc826c",
15:12:59         "io.kubernetes.cri-o.HostnamePath": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/hostname",
15:12:59         "io.kubernetes.cri-o.ContainerName": "k8s_POD_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1",
15:12:59         "io.kubernetes.cri-o.PrivilegedRuntime": "false",
15:12:59         "io.kubernetes.cri-o.ResolvPath": "",
15:12:59         "io.kubernetes.cri-o.MountPoint": "/var/lib/containers/storage/overlay/e082e3f9c46e18e50e53b6a49c1739cfd029452c469f07a7937794a5a50b9db0/merged",
15:12:59         "io.kubernetes.cri-o.PortMappings": "[]",
15:12:59         "io.kubernetes.cri-o.CNIResult": "{\"cniVersion\":\"0.4.0\",\"interfaces\":[{\"name\":\"cni0\",\"mac\":\"ae:b0:19:cf:37:45\"},{\"name\":\"veth2325b942\",\"mac\":\"ba:dc:1e:4e:05:f6\"},{\"name\":\"eth0\",\"mac\":\"da:81:56:36:25:34\",\"sandbox\":\"/var/run/netns/0cc4fc54-1d0d-460e-8509-822548ce4a9c\"}],\"ips\":[{\"version\":\"4\",\"interface\":2,\"address\":\"10.244.0.2/24\",\"gateway\":\"10.244.0.1\"}],\"routes\":[{\"dst\":\"10.244.0.0/16\"},{\"dst\":\"0.0.0.0/0\",\"gw\":\"10.244.0.1\"}],\"dns\":{}}",
15:12:59         "io.kubernetes.cri-o.Metadata": "{\"Name\":\"nginx-sandbox\",\"UID\":\"hdishd83djaidwnduwk28bcs3\",\"Namespace\":\"default\",\"Attempt\":1}",
15:12:59         "io.kubernetes.cri-o.Namespace": "default",
15:12:59         "io.kubernetes.cri-o.SandboxID": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:12:59         "io.kubernetes.cri-o.KubeName": "nginx-sandbox",
15:12:59         "io.container.manager": "cri-o",
15:12:59         "io.kubernetes.cri-o.Name": "k8s_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1",
15:12:59         "io.kubernetes.cri-o.ContainerType": "sandbox",
15:12:59         "io.kubernetes.cri-o.ShmPath": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/shm",
15:12:59         "io.kubernetes.cri-o.CgroupParent": ""
15:12:59       },
15:12:59       "linux": {
15:12:59         "resources": {
15:12:59           "devices": [
15:12:59             {
15:12:59               "allow": false,
15:12:59               "access": "rwm"
15:12:59             }
15:12:59           ],
15:12:59           "cpu": {
15:12:59             "shares": 2
15:12:59           }
15:12:59         },
15:12:59         "namespaces": [
15:12:59           {
15:12:59             "type": "pid"
15:12:59           },
15:12:59           {
15:12:59             "type": "network",
15:12:59             "path": "/var/run/netns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:12:59           },
15:12:59           {
15:12:59             "type": "ipc",
15:12:59             "path": "/var/run/ipcns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:12:59           },
15:12:59           {
15:12:59             "type": "uts",
15:12:59             "path": "/var/run/utsns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:12:59           },
15:12:59           {
15:12:59             "type": "mount"
15:12:59           }
15:12:59         ],
15:12:59         "mountLabel": "system_u:object_r:container_file_t:s0:c288,c583"
15:12:59       }
15:12:59     }
15:12:59   }
15:12:59 }
15:12:59 /tmp/jenkins/workspace/kata-containers-2.0-tests-centos-PR/go/src/github.com/kata-containers/tests/lib/common.bash: line 61: 8e56e4946042fbe5dce8609ea349724e68f1a70eb0d7d2839a32f07787bc236c: command not found
15:13:02 {
15:13:02   "status": {
15:13:02     "id": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:13:02     "metadata": {
15:13:02       "attempt": 1,
15:13:02       "name": "nginx-sandbox",
15:13:02       "namespace": "default",
15:13:02       "uid": "hdishd83djaidwnduwk28bcs3"
15:13:02     },
15:13:02     "state": "SANDBOX_READY",
15:13:02     "createdAt": "2021-10-25T18:12:52.859870345Z",
15:13:02     "network": {
15:13:02       "ip": "10.244.0.2"
15:13:02     },
15:13:02     "linux": {
15:13:02       "namespaces": {
15:13:02         "options": {
15:13:02           "ipc": "POD",
15:13:02           "network": "POD",
15:13:02           "pid": "POD"
15:13:02         }
15:13:02       }
15:13:02     },
15:13:02     "labels": {},
15:13:02     "annotations": {}
15:13:02   },
15:13:02   "info": {
15:13:02     "image": "k8s.gcr.io/pause:3.5",
15:13:02     "pid": 108390,
15:13:02     "runtimeSpec": {
15:13:02       "ociVersion": "1.0.2-dev",
15:13:02       "process": {
15:13:02         "user": {
15:13:02           "uid": 0,
15:13:02           "gid": 0
15:13:02         },
15:13:02         "args": [
15:13:02           "/pause"
15:13:02         ],
15:13:02         "env": [
15:13:02           "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
15:13:02           "TERM=xterm"
15:13:02         ],
15:13:02         "cwd": "/",
15:13:02         "capabilities": {
15:13:02           "bounding": [
15:13:02             "CAP_CHOWN",
15:13:02             "CAP_DAC_OVERRIDE",
15:13:02             "CAP_FSETID",
15:13:02             "CAP_FOWNER",
15:13:02             "CAP_SETGID",
15:13:02             "CAP_SETUID",
15:13:02             "CAP_SETPCAP",
15:13:02             "CAP_NET_BIND_SERVICE",
15:13:02             "CAP_KILL"
15:13:02           ],
15:13:02           "effective": [
15:13:02             "CAP_CHOWN",
15:13:02             "CAP_DAC_OVERRIDE",
15:13:02             "CAP_FSETID",
15:13:02             "CAP_FOWNER",
15:13:02             "CAP_SETGID",
15:13:02             "CAP_SETUID",
15:13:02             "CAP_SETPCAP",
15:13:02             "CAP_NET_BIND_SERVICE",
15:13:02             "CAP_KILL"
15:13:02           ],
15:13:02           "inheritable": [
15:13:02             "CAP_CHOWN",
15:13:02             "CAP_DAC_OVERRIDE",
15:13:02             "CAP_FSETID",
15:13:02             "CAP_FOWNER",
15:13:02             "CAP_SETGID",
15:13:02             "CAP_SETUID",
15:13:02             "CAP_SETPCAP",
15:13:02             "CAP_NET_BIND_SERVICE",
15:13:02             "CAP_KILL"
15:13:02           ],
15:13:02           "permitted": [
15:13:02             "CAP_CHOWN",
15:13:02             "CAP_DAC_OVERRIDE",
15:13:02             "CAP_FSETID",
15:13:02             "CAP_FOWNER",
15:13:02             "CAP_SETGID",
15:13:02             "CAP_SETUID",
15:13:02             "CAP_SETPCAP",
15:13:02             "CAP_NET_BIND_SERVICE",
15:13:02             "CAP_KILL"
15:13:02           ]
15:13:02         },
15:13:02         "oomScoreAdj": -998,
15:13:02         "selinuxLabel": "system_u:system_r:container_kvm_t:s0:c288,c583"
15:13:02       },
15:13:02       "root": {
15:13:02         "path": "/var/lib/containers/storage/overlay/e082e3f9c46e18e50e53b6a49c1739cfd029452c469f07a7937794a5a50b9db0/merged",
15:13:02         "readonly": true
15:13:02       },
15:13:02       "hostname": "41f9b2fc826c",
15:13:02       "mounts": [
15:13:02         {
15:13:02           "destination": "/proc",
15:13:02           "type": "proc",
15:13:02           "source": "proc",
15:13:02           "options": [
15:13:02             "nosuid",
15:13:02             "noexec",
15:13:02             "nodev"
15:13:02           ]
15:13:02         },
15:13:02         {
15:13:02           "destination": "/dev",
15:13:02           "type": "tmpfs",
15:13:02           "source": "tmpfs",
15:13:02           "options": [
15:13:02             "nosuid",
15:13:02             "strictatime",
15:13:02             "mode=755",
15:13:02             "size=65536k"
15:13:02           ]
15:13:02         },
15:13:02         {
15:13:02           "destination": "/dev/pts",
15:13:02           "type": "devpts",
15:13:02           "source": "devpts",
15:13:02           "options": [
15:13:02             "nosuid",
15:13:02             "noexec",
15:13:02             "newinstance",
15:13:02             "ptmxmode=0666",
15:13:02             "mode=0620",
15:13:02             "gid=5"
15:13:02           ]
15:13:02         },
15:13:02         {
15:13:02           "destination": "/dev/mqueue",
15:13:02           "type": "mqueue",
15:13:02           "source": "mqueue",
15:13:02           "options": [
15:13:02             "nosuid",
15:13:02             "noexec",
15:13:02             "nodev"
15:13:02           ]
15:13:02         },
15:13:02         {
15:13:02           "destination": "/sys",
15:13:02           "type": "sysfs",
15:13:02           "source": "sysfs",
15:13:02           "options": [
15:13:02             "nosuid",
15:13:02             "noexec",
15:13:02             "nodev",
15:13:02             "ro"
15:13:02           ]
15:13:02         },
15:13:02         {
15:13:02           "destination": "/dev/shm",
15:13:02           "type": "bind",
15:13:02           "source": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/shm",
15:13:02           "options": [
15:13:02             "rw",
15:13:02             "bind"
15:13:02           ]
15:13:02         },
15:13:02         {
15:13:02           "destination": "/etc/hostname",
15:13:02           "type": "bind",
15:13:02           "source": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/hostname",
15:13:02           "options": [
15:13:02             "ro",
15:13:02             "bind",
15:13:02             "nodev",
15:13:02             "nosuid",
15:13:02             "noexec"
15:13:02           ]
15:13:02         }
15:13:02       ],
15:13:02       "annotations": {
15:13:02         "io.kubernetes.cri-o.Created": "2021-10-25T18:12:52.859870345Z",
15:13:02         "io.kubernetes.cri-o.SeccompProfilePath": "",
15:13:02         "io.kubernetes.cri-o.LogPath": "/tmp/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24.log",
15:13:02         "io.kubernetes.cri-o.NamespaceOptions": "{}",
15:13:02         "io.kubernetes.cri-o.Annotations": "null",
15:13:02         "org.systemd.property.CollectMode": "'inactive-or-failed'",
15:13:02         "io.kubernetes.cri-o.ContainerID": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:13:02         "io.kubernetes.cri-o.RuntimeHandler": "kata",
15:13:02         "io.kubernetes.cri-o.HostNetwork": "false",
15:13:02         "io.kubernetes.cri-o.IP.0": "10.244.0.2",
15:13:02         "io.kubernetes.cri-o.Labels": "null",
15:13:02         "io.kubernetes.cri-o.Image": "k8s.gcr.io/pause:3.5",
15:13:02         "io.kubernetes.cri-o.HostName": "41f9b2fc826c",
15:13:02         "io.kubernetes.cri-o.HostnamePath": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/hostname",
15:13:02         "io.kubernetes.cri-o.ResolvPath": "",
15:13:02         "io.kubernetes.cri-o.MountPoint": "/var/lib/containers/storage/overlay/e082e3f9c46e18e50e53b6a49c1739cfd029452c469f07a7937794a5a50b9db0/merged",
15:13:02         "io.kubernetes.cri-o.ContainerName": "k8s_POD_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1",
15:13:02         "io.kubernetes.cri-o.PrivilegedRuntime": "false",
15:13:02         "io.kubernetes.cri-o.SandboxID": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:13:02         "io.kubernetes.cri-o.KubeName": "nginx-sandbox",
15:13:02         "io.container.manager": "cri-o",
15:13:02         "io.kubernetes.cri-o.PortMappings": "[]",
15:13:02         "io.kubernetes.cri-o.CNIResult": "{\"cniVersion\":\"0.4.0\",\"interfaces\":[{\"name\":\"cni0\",\"mac\":\"ae:b0:19:cf:37:45\"},{\"name\":\"veth2325b942\",\"mac\":\"ba:dc:1e:4e:05:f6\"},{\"name\":\"eth0\",\"mac\":\"da:81:56:36:25:34\",\"sandbox\":\"/var/run/netns/0cc4fc54-1d0d-460e-8509-822548ce4a9c\"}],\"ips\":[{\"version\":\"4\",\"interface\":2,\"address\":\"10.244.0.2/24\",\"gateway\":\"10.244.0.1\"}],\"routes\":[{\"dst\":\"10.244.0.0/16\"},{\"dst\":\"0.0.0.0/0\",\"gw\":\"10.244.0.1\"}],\"dns\":{}}",
15:13:02         "io.kubernetes.cri-o.Metadata": "{\"Name\":\"nginx-sandbox\",\"UID\":\"hdishd83djaidwnduwk28bcs3\",\"Namespace\":\"default\",\"Attempt\":1}",
15:13:02         "io.kubernetes.cri-o.Namespace": "default",
15:13:02         "io.kubernetes.cri-o.ShmPath": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/shm",
15:13:02         "io.kubernetes.cri-o.CgroupParent": "",
15:13:02         "io.kubernetes.cri-o.Name": "k8s_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1",
15:13:02         "io.kubernetes.cri-o.ContainerType": "sandbox"
15:13:02       },
15:13:02       "linux": {
15:13:02         "resources": {
15:13:02           "devices": [
15:13:02             {
15:13:02               "allow": false,
15:13:02               "access": "rwm"
15:13:02             }
15:13:02           ],
15:13:02           "cpu": {
15:13:02             "shares": 2
15:13:02           }
15:13:02         },
15:13:02         "namespaces": [
15:13:02           {
15:13:02             "type": "pid"
15:13:02           },
15:13:02           {
15:13:02             "type": "network",
15:13:02             "path": "/var/run/netns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:13:02           },
15:13:02           {
15:13:02             "type": "ipc",
15:13:02             "path": "/var/run/ipcns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:13:02           },
15:13:02           {
15:13:02             "type": "uts",
15:13:02             "path": "/var/run/utsns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:13:02           },
15:13:02           {
15:13:02             "type": "mount"
15:13:02           }
15:13:02         ],
15:13:02         "mountLabel": "system_u:object_r:container_file_t:s0:c288,c583"
15:13:02       }
15:13:02     }
15:13:02   }
15:13:02 }
15:13:02 /tmp/jenkins/workspace/kata-containers-2.0-tests-centos-PR/go/src/github.com/kata-containers/tests/lib/common.bash: line 61: 8e56e4946042fbe5dce8609ea349724e68f1a70eb0d7d2839a32f07787bc236c: command not found
15:13:05 {
15:13:05   "status": {
15:13:05     "id": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:13:05     "metadata": {
15:13:05       "attempt": 1,
15:13:05       "name": "nginx-sandbox",
15:13:05       "namespace": "default",
15:13:05       "uid": "hdishd83djaidwnduwk28bcs3"
15:13:05     },
15:13:05     "state": "SANDBOX_READY",
15:13:05     "createdAt": "2021-10-25T18:12:52.859870345Z",
15:13:05     "network": {
15:13:05       "ip": "10.244.0.2"
15:13:05     },
15:13:05     "linux": {
15:13:05       "namespaces": {
15:13:05         "options": {
15:13:05           "ipc": "POD",
15:13:05           "network": "POD",
15:13:05           "pid": "POD"
15:13:05         }
15:13:05       }
15:13:05     },
15:13:05     "labels": {},
15:13:05     "annotations": {}
15:13:05   },
15:13:05   "info": {
15:13:05     "image": "k8s.gcr.io/pause:3.5",
15:13:05     "pid": 108390,
15:13:05     "runtimeSpec": {
15:13:05       "ociVersion": "1.0.2-dev",
15:13:05       "process": {
15:13:05         "user": {
15:13:05           "uid": 0,
15:13:05           "gid": 0
15:13:05         },
15:13:05         "args": [
15:13:05           "/pause"
15:13:05         ],
15:13:05         "env": [
15:13:05           "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
15:13:05           "TERM=xterm"
15:13:05         ],
15:13:05         "cwd": "/",
15:13:05         "capabilities": {
15:13:05           "bounding": [
15:13:05             "CAP_CHOWN",
15:13:05             "CAP_DAC_OVERRIDE",
15:13:05             "CAP_FSETID",
15:13:05             "CAP_FOWNER",
15:13:05             "CAP_SETGID",
15:13:05             "CAP_SETUID",
15:13:05             "CAP_SETPCAP",
15:13:05             "CAP_NET_BIND_SERVICE",
15:13:05             "CAP_KILL"
15:13:05           ],
15:13:05           "effective": [
15:13:05             "CAP_CHOWN",
15:13:05             "CAP_DAC_OVERRIDE",
15:13:05             "CAP_FSETID",
15:13:05             "CAP_FOWNER",
15:13:05             "CAP_SETGID",
15:13:05             "CAP_SETUID",
15:13:05             "CAP_SETPCAP",
15:13:05             "CAP_NET_BIND_SERVICE",
15:13:05             "CAP_KILL"
15:13:05           ],
15:13:05           "inheritable": [
15:13:05             "CAP_CHOWN",
15:13:05             "CAP_DAC_OVERRIDE",
15:13:05             "CAP_FSETID",
15:13:05             "CAP_FOWNER",
15:13:05             "CAP_SETGID",
15:13:05             "CAP_SETUID",
15:13:05             "CAP_SETPCAP",
15:13:05             "CAP_NET_BIND_SERVICE",
15:13:05             "CAP_KILL"
15:13:05           ],
15:13:05           "permitted": [
15:13:05             "CAP_CHOWN",
15:13:05             "CAP_DAC_OVERRIDE",
15:13:05             "CAP_FSETID",
15:13:05             "CAP_FOWNER",
15:13:05             "CAP_SETGID",
15:13:05             "CAP_SETUID",
15:13:05             "CAP_SETPCAP",
15:13:05             "CAP_NET_BIND_SERVICE",
15:13:05             "CAP_KILL"
15:13:05           ]
15:13:05         },
15:13:05         "oomScoreAdj": -998,
15:13:05         "selinuxLabel": "system_u:system_r:container_kvm_t:s0:c288,c583"
15:13:05       },
15:13:05       "root": {
15:13:05         "path": "/var/lib/containers/storage/overlay/e082e3f9c46e18e50e53b6a49c1739cfd029452c469f07a7937794a5a50b9db0/merged",
15:13:05         "readonly": true
15:13:05       },
15:13:05       "hostname": "41f9b2fc826c",
15:13:05       "mounts": [
15:13:05         {
15:13:05           "destination": "/proc",
15:13:05           "type": "proc",
15:13:05           "source": "proc",
15:13:05           "options": [
15:13:05             "nosuid",
15:13:05             "noexec",
15:13:05             "nodev"
15:13:05           ]
15:13:05         },
15:13:05         {
15:13:05           "destination": "/dev",
15:13:05           "type": "tmpfs",
15:13:05           "source": "tmpfs",
15:13:05           "options": [
15:13:05             "nosuid",
15:13:05             "strictatime",
15:13:05             "mode=755",
15:13:05             "size=65536k"
15:13:05           ]
15:13:05         },
15:13:05         {
15:13:05           "destination": "/dev/pts",
15:13:05           "type": "devpts",
15:13:05           "source": "devpts",
15:13:05           "options": [
15:13:05             "nosuid",
15:13:05             "noexec",
15:13:05             "newinstance",
15:13:05             "ptmxmode=0666",
15:13:05             "mode=0620",
15:13:05             "gid=5"
15:13:05           ]
15:13:05         },
15:13:05         {
15:13:05           "destination": "/dev/mqueue",
15:13:05           "type": "mqueue",
15:13:05           "source": "mqueue",
15:13:05           "options": [
15:13:05             "nosuid",
15:13:05             "noexec",
15:13:05             "nodev"
15:13:05           ]
15:13:05         },
15:13:05         {
15:13:05           "destination": "/sys",
15:13:05           "type": "sysfs",
15:13:05           "source": "sysfs",
15:13:05           "options": [
15:13:05             "nosuid",
15:13:05             "noexec",
15:13:05             "nodev",
15:13:05             "ro"
15:13:05           ]
15:13:05         },
15:13:05         {
15:13:05           "destination": "/dev/shm",
15:13:05           "type": "bind",
15:13:05           "source": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/shm",
15:13:05           "options": [
15:13:05             "rw",
15:13:05             "bind"
15:13:05           ]
15:13:05         },
15:13:05         {
15:13:05           "destination": "/etc/hostname",
15:13:05           "type": "bind",
15:13:05           "source": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/hostname",
15:13:05           "options": [
15:13:05             "ro",
15:13:05             "bind",
15:13:05             "nodev",
15:13:05             "nosuid",
15:13:05             "noexec"
15:13:05           ]
15:13:05         }
15:13:05       ],
15:13:05       "annotations": {
15:13:05         "io.kubernetes.cri-o.HostName": "41f9b2fc826c",
15:13:05         "io.kubernetes.cri-o.HostnamePath": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/hostname",
15:13:05         "io.kubernetes.cri-o.ContainerName": "k8s_POD_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1",
15:13:05         "io.kubernetes.cri-o.PrivilegedRuntime": "false",
15:13:05         "io.kubernetes.cri-o.ResolvPath": "",
15:13:05         "io.kubernetes.cri-o.MountPoint": "/var/lib/containers/storage/overlay/e082e3f9c46e18e50e53b6a49c1739cfd029452c469f07a7937794a5a50b9db0/merged",
15:13:05         "io.kubernetes.cri-o.Metadata": "{\"Name\":\"nginx-sandbox\",\"UID\":\"hdishd83djaidwnduwk28bcs3\",\"Namespace\":\"default\",\"Attempt\":1}",
15:13:05         "io.kubernetes.cri-o.Namespace": "default",
15:13:05         "io.kubernetes.cri-o.SandboxID": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:13:05         "io.kubernetes.cri-o.KubeName": "nginx-sandbox",
15:13:05         "io.container.manager": "cri-o",
15:13:05         "io.kubernetes.cri-o.PortMappings": "[]",
15:13:05         "io.kubernetes.cri-o.CNIResult": "{\"cniVersion\":\"0.4.0\",\"interfaces\":[{\"name\":\"cni0\",\"mac\":\"ae:b0:19:cf:37:45\"},{\"name\":\"veth2325b942\",\"mac\":\"ba:dc:1e:4e:05:f6\"},{\"name\":\"eth0\",\"mac\":\"da:81:56:36:25:34\",\"sandbox\":\"/var/run/netns/0cc4fc54-1d0d-460e-8509-822548ce4a9c\"}],\"ips\":[{\"version\":\"4\",\"interface\":2,\"address\":\"10.244.0.2/24\",\"gateway\":\"10.244.0.1\"}],\"routes\":[{\"dst\":\"10.244.0.0/16\"},{\"dst\":\"0.0.0.0/0\",\"gw\":\"10.244.0.1\"}],\"dns\":{}}",
15:13:05         "io.kubernetes.cri-o.Name": "k8s_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1",
15:13:05         "io.kubernetes.cri-o.ContainerType": "sandbox",
15:13:05         "io.kubernetes.cri-o.ShmPath": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/shm",
15:13:05         "io.kubernetes.cri-o.CgroupParent": "",
15:13:05         "io.kubernetes.cri-o.LogPath": "/tmp/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24.log",
15:13:05         "io.kubernetes.cri-o.NamespaceOptions": "{}",
15:13:05         "io.kubernetes.cri-o.Created": "2021-10-25T18:12:52.859870345Z",
15:13:05         "io.kubernetes.cri-o.SeccompProfilePath": "",
15:13:05         "io.kubernetes.cri-o.Annotations": "null",
15:13:05         "org.systemd.property.CollectMode": "'inactive-or-failed'",
15:13:05         "io.kubernetes.cri-o.Labels": "null",
15:13:05         "io.kubernetes.cri-o.Image": "k8s.gcr.io/pause:3.5",
15:13:05         "io.kubernetes.cri-o.ContainerID": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:13:05         "io.kubernetes.cri-o.RuntimeHandler": "kata",
15:13:05         "io.kubernetes.cri-o.HostNetwork": "false",
15:13:05         "io.kubernetes.cri-o.IP.0": "10.244.0.2"
15:13:05       },
15:13:05       "linux": {
15:13:05         "resources": {
15:13:05           "devices": [
15:13:05             {
15:13:05               "allow": false,
15:13:05               "access": "rwm"
15:13:05             }
15:13:05           ],
15:13:05           "cpu": {
15:13:05             "shares": 2
15:13:05           }
15:13:05         },
15:13:05         "namespaces": [
15:13:05           {
15:13:05             "type": "pid"
15:13:05           },
15:13:05           {
15:13:05             "type": "network",
15:13:05             "path": "/var/run/netns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:13:05           },
15:13:05           {
15:13:05             "type": "ipc",
15:13:05             "path": "/var/run/ipcns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:13:05           },
15:13:05           {
15:13:05             "type": "uts",
15:13:05             "path": "/var/run/utsns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:13:05           },
15:13:05           {
15:13:05             "type": "mount"
15:13:05           }
15:13:05         ],
15:13:05         "mountLabel": "system_u:object_r:container_file_t:s0:c288,c583"
15:13:05       }
15:13:05     }
15:13:05   }
15:13:05 }
15:13:05 /tmp/jenkins/workspace/kata-containers-2.0-tests-centos-PR/go/src/github.com/kata-containers/tests/lib/common.bash: line 61: 8e56e4946042fbe5dce8609ea349724e68f1a70eb0d7d2839a32f07787bc236c: command not found
15:13:08 {
15:13:08   "status": {
15:13:08     "id": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:13:08     "metadata": {
15:13:08       "attempt": 1,
15:13:08       "name": "nginx-sandbox",
15:13:08       "namespace": "default",
15:13:08       "uid": "hdishd83djaidwnduwk28bcs3"
15:13:08     },
15:13:08     "state": "SANDBOX_READY",
15:13:08     "createdAt": "2021-10-25T18:12:52.859870345Z",
15:13:08     "network": {
15:13:08       "ip": "10.244.0.2"
15:13:08     },
15:13:08     "linux": {
15:13:08       "namespaces": {
15:13:08         "options": {
15:13:08           "ipc": "POD",
15:13:08           "network": "POD",
15:13:08           "pid": "POD"
15:13:08         }
15:13:08       }
15:13:08     },
15:13:08     "labels": {},
15:13:08     "annotations": {}
15:13:08   },
15:13:08   "info": {
15:13:08     "image": "k8s.gcr.io/pause:3.5",
15:13:08     "pid": 108390,
15:13:08     "runtimeSpec": {
15:13:08       "ociVersion": "1.0.2-dev",
15:13:08       "process": {
15:13:08         "user": {
15:13:08           "uid": 0,
15:13:08           "gid": 0
15:13:08         },
15:13:08         "args": [
15:13:08           "/pause"
15:13:08         ],
15:13:08         "env": [
15:13:08           "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
15:13:08           "TERM=xterm"
15:13:08         ],
15:13:08         "cwd": "/",
15:13:08         "capabilities": {
15:13:08           "bounding": [
15:13:08             "CAP_CHOWN",
15:13:08             "CAP_DAC_OVERRIDE",
15:13:08             "CAP_FSETID",
15:13:08             "CAP_FOWNER",
15:13:08             "CAP_SETGID",
15:13:08             "CAP_SETUID",
15:13:08             "CAP_SETPCAP",
15:13:08             "CAP_NET_BIND_SERVICE",
15:13:08             "CAP_KILL"
15:13:08           ],
15:13:08           "effective": [
15:13:08             "CAP_CHOWN",
15:13:08             "CAP_DAC_OVERRIDE",
15:13:08             "CAP_FSETID",
15:13:08             "CAP_FOWNER",
15:13:08             "CAP_SETGID",
15:13:08             "CAP_SETUID",
15:13:08             "CAP_SETPCAP",
15:13:08             "CAP_NET_BIND_SERVICE",
15:13:08             "CAP_KILL"
15:13:08           ],
15:13:08           "inheritable": [
15:13:08             "CAP_CHOWN",
15:13:08             "CAP_DAC_OVERRIDE",
15:13:08             "CAP_FSETID",
15:13:08             "CAP_FOWNER",
15:13:08             "CAP_SETGID",
15:13:08             "CAP_SETUID",
15:13:08             "CAP_SETPCAP",
15:13:08             "CAP_NET_BIND_SERVICE",
15:13:08             "CAP_KILL"
15:13:08           ],
15:13:08           "permitted": [
15:13:08             "CAP_CHOWN",
15:13:08             "CAP_DAC_OVERRIDE",
15:13:08             "CAP_FSETID",
15:13:08             "CAP_FOWNER",
15:13:08             "CAP_SETGID",
15:13:08             "CAP_SETUID",
15:13:08             "CAP_SETPCAP",
15:13:08             "CAP_NET_BIND_SERVICE",
15:13:08             "CAP_KILL"
15:13:08           ]
15:13:08         },
15:13:08         "oomScoreAdj": -998,
15:13:08         "selinuxLabel": "system_u:system_r:container_kvm_t:s0:c288,c583"
15:13:08       },
15:13:08       "root": {
15:13:08         "path": "/var/lib/containers/storage/overlay/e082e3f9c46e18e50e53b6a49c1739cfd029452c469f07a7937794a5a50b9db0/merged",
15:13:08         "readonly": true
15:13:08       },
15:13:08       "hostname": "41f9b2fc826c",
15:13:08       "mounts": [
15:13:08         {
15:13:08           "destination": "/proc",
15:13:08           "type": "proc",
15:13:08           "source": "proc",
15:13:08           "options": [
15:13:08             "nosuid",
15:13:08             "noexec",
15:13:08             "nodev"
15:13:08           ]
15:13:08         },
15:13:08         {
15:13:08           "destination": "/dev",
15:13:08           "type": "tmpfs",
15:13:08           "source": "tmpfs",
15:13:08           "options": [
15:13:08             "nosuid",
15:13:08             "strictatime",
15:13:08             "mode=755",
15:13:08             "size=65536k"
15:13:08           ]
15:13:08         },
15:13:08         {
15:13:08           "destination": "/dev/pts",
15:13:08           "type": "devpts",
15:13:08           "source": "devpts",
15:13:08           "options": [
15:13:08             "nosuid",
15:13:08             "noexec",
15:13:08             "newinstance",
15:13:08             "ptmxmode=0666",
15:13:08             "mode=0620",
15:13:08             "gid=5"
15:13:08           ]
15:13:08         },
15:13:08         {
15:13:08           "destination": "/dev/mqueue",
15:13:08           "type": "mqueue",
15:13:08           "source": "mqueue",
15:13:08           "options": [
15:13:08             "nosuid",
15:13:08             "noexec",
15:13:08             "nodev"
15:13:08           ]
15:13:08         },
15:13:08         {
15:13:08           "destination": "/sys",
15:13:08           "type": "sysfs",
15:13:08           "source": "sysfs",
15:13:08           "options": [
15:13:08             "nosuid",
15:13:08             "noexec",
15:13:08             "nodev",
15:13:08             "ro"
15:13:08           ]
15:13:08         },
15:13:08         {
15:13:08           "destination": "/dev/shm",
15:13:08           "type": "bind",
15:13:08           "source": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/shm",
15:13:08           "options": [
15:13:08             "rw",
15:13:08             "bind"
15:13:08           ]
15:13:08         },
15:13:08         {
15:13:08           "destination": "/etc/hostname",
15:13:08           "type": "bind",
15:13:08           "source": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/hostname",
15:13:08           "options": [
15:13:08             "ro",
15:13:08             "bind",
15:13:08             "nodev",
15:13:08             "nosuid",
15:13:08             "noexec"
15:13:08           ]
15:13:08         }
15:13:08       ],
15:13:08       "annotations": {
15:13:08         "io.container.manager": "cri-o",
15:13:08         "io.kubernetes.cri-o.PortMappings": "[]",
15:13:08         "io.kubernetes.cri-o.CNIResult": "{\"cniVersion\":\"0.4.0\",\"interfaces\":[{\"name\":\"cni0\",\"mac\":\"ae:b0:19:cf:37:45\"},{\"name\":\"veth2325b942\",\"mac\":\"ba:dc:1e:4e:05:f6\"},{\"name\":\"eth0\",\"mac\":\"da:81:56:36:25:34\",\"sandbox\":\"/var/run/netns/0cc4fc54-1d0d-460e-8509-822548ce4a9c\"}],\"ips\":[{\"version\":\"4\",\"interface\":2,\"address\":\"10.244.0.2/24\",\"gateway\":\"10.244.0.1\"}],\"routes\":[{\"dst\":\"10.244.0.0/16\"},{\"dst\":\"0.0.0.0/0\",\"gw\":\"10.244.0.1\"}],\"dns\":{}}",
15:13:08         "io.kubernetes.cri-o.Metadata": "{\"Name\":\"nginx-sandbox\",\"UID\":\"hdishd83djaidwnduwk28bcs3\",\"Namespace\":\"default\",\"Attempt\":1}",
15:13:08         "io.kubernetes.cri-o.Namespace": "default",
15:13:08         "io.kubernetes.cri-o.SandboxID": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:13:08         "io.kubernetes.cri-o.KubeName": "nginx-sandbox",
15:13:08         "io.kubernetes.cri-o.Name": "k8s_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1",
15:13:08         "io.kubernetes.cri-o.ContainerType": "sandbox",
15:13:08         "io.kubernetes.cri-o.ShmPath": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/shm",
15:13:08         "io.kubernetes.cri-o.CgroupParent": "",
15:13:08         "io.kubernetes.cri-o.LogPath": "/tmp/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24.log",
15:13:08         "io.kubernetes.cri-o.NamespaceOptions": "{}",
15:13:08         "io.kubernetes.cri-o.Created": "2021-10-25T18:12:52.859870345Z",
15:13:08         "io.kubernetes.cri-o.SeccompProfilePath": "",
15:13:08         "io.kubernetes.cri-o.Annotations": "null",
15:13:08         "org.systemd.property.CollectMode": "'inactive-or-failed'",
15:13:08         "io.kubernetes.cri-o.HostNetwork": "false",
15:13:08         "io.kubernetes.cri-o.IP.0": "10.244.0.2",
15:13:08         "io.kubernetes.cri-o.Labels": "null",
15:13:08         "io.kubernetes.cri-o.Image": "k8s.gcr.io/pause:3.5",
15:13:08         "io.kubernetes.cri-o.ContainerID": "41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24",
15:13:08         "io.kubernetes.cri-o.RuntimeHandler": "kata",
15:13:08         "io.kubernetes.cri-o.HostName": "41f9b2fc826c",
15:13:08         "io.kubernetes.cri-o.HostnamePath": "/run/containers/storage/overlay-containers/41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24/userdata/hostname",
15:13:08         "io.kubernetes.cri-o.ContainerName": "k8s_POD_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1",
15:13:08         "io.kubernetes.cri-o.PrivilegedRuntime": "false",
15:13:08         "io.kubernetes.cri-o.ResolvPath": "",
15:13:08         "io.kubernetes.cri-o.MountPoint": "/var/lib/containers/storage/overlay/e082e3f9c46e18e50e53b6a49c1739cfd029452c469f07a7937794a5a50b9db0/merged"
15:13:08       },
15:13:08       "linux": {
15:13:08         "resources": {
15:13:08           "devices": [
15:13:08             {
15:13:08               "allow": false,
15:13:08               "access": "rwm"
15:13:08             }
15:13:08           ],
15:13:08           "cpu": {
15:13:08             "shares": 2
15:13:08           }
15:13:08         },
15:13:08         "namespaces": [
15:13:08           {
15:13:08             "type": "pid"
15:13:08           },
15:13:08           {
15:13:08             "type": "network",
15:13:08             "path": "/var/run/netns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:13:08           },
15:13:08           {
15:13:08             "type": "ipc",
15:13:08             "path": "/var/run/ipcns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:13:08           },
15:13:08           {
15:13:08             "type": "uts",
15:13:08             "path": "/var/run/utsns/0cc4fc54-1d0d-460e-8509-822548ce4a9c"
15:13:08           },
15:13:08           {
15:13:08             "type": "mount"
15:13:08           }
15:13:08         ],
15:13:08         "mountLabel": "system_u:object_r:container_file_t:s0:c288,c583"
15:13:08       }
15:13:08     }
15:13:08   }
15:13:08 }
15:13:08 /tmp/jenkins/workspace/kata-containers-2.0-tests-centos-PR/go/src/github.com/kata-containers/tests/lib/common.bash: line 61: 8e56e4946042fbe5dce8609ea349724e68f1a70eb0d7d2839a32f07787bc236c: command not found
15:13:11 [rootless_test.sh:35] ERROR: return 1
15:13:12 Stopped sandbox 41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24
15:13:12 Stopped sandbox 8e56e4946042fbe5dce8609ea349724e68f1a70eb0d7d2839a32f07787bc236c
15:13:12 Removed sandbox 41f9b2fc826c9e95301196db74e0c1b1ee06eb91471fd4fb2864e5c20b592b24
15:13:12 Removed sandbox 8e56e4946042fbe5dce8609ea349724e68f1a70eb0d7d2839a32f07787bc236c
15:13:12 make: *** [Makefile:177: rootless] Error 1
15:13:12 [run.sh:72] ERROR: sudo -E PATH=/tmp/jenkins/workspace/kata-containers-2.0-tests-centos-PR/go/bin:/usr/local/go/bin:/usr/sbin:/sbin:/home/jenkins/.local/bin:/home/jenkins/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin bash -c make rootless
```

--- a/integration/rootless/rootless_test.sh
+++ b/integration/rootless/rootless_test.sh
@@ -14,6 +14,7 @@ dir_path=$(dirname "$0")
 source "${dir_path}/../../lib/common.bash"
 source "${dir_path}/../../.ci/lib.sh"
 source /etc/os-release || source /usr/lib/os-release
+pod_id=""
 
 setup() {
 	getent group kvm &>/dev/null || sudo groupadd --system kvm
@@ -26,13 +27,13 @@ setup() {
 
 cleanup() {
 	sudo sed -i -e 's/^.*\(rootless\)/# \1/g' /usr/share/defaults/kata-containers/configuration.toml
-	sudo crictl stopp $(sudo crictl pods -q) || true
-	sudo crictl rmp $(sudo crictl pods -q) || true
+	sudo crictl stopp "$pod_id" || true
+	sudo crictl rmp "$pod_id" || true
 }
 
 run() {
-	sudo crictl runp -r kata "${dir_path}/rootless-pod.json"
-	waitForProcess 15 3 "sudo crictl inspectp $(sudo crictl pods -q) | jq '.status.state' | grep 'SANDBOX_READY'"
+	pod_id="$(sudo crictl runp -r kata "${dir_path}/rootless-pod.json")"
+	waitForProcess 15 3 "sudo crictl inspectp "$pod_id" | jq '.status.state' | grep 'SANDBOX_READY'"
 }
 
 main() {

